test/LightDFGSummaryTypes.jl

```diff
@@ -112,58 +117,60 @@ end
     # @test data(f1) == f1.data
     # @test getData(f1) == f1.data
     # Internal function
-    @test internalId(f1) == f1._internalId
+    if FACTYPE == DFGFactorSummary
+        @test internalId(f1) == f1._internalId
+    end
 
-    @test getSolverParams(dfg) != nothing
-    @test setSolverParams(dfg, getSolverParams(dfg)) == getSolverParams(dfg)
 end
 
 @testset "Updating Nodes" begin
-    global dfg
-    #get the variable
-    var = getVariable(dfg, :a)
-    #make a copy and simulate external changes
-    newvar = deepcopy(var)
-    estimates(newvar)[:default] = Dict{Symbol, VariableEstimate}(
-        :max => VariableEstimate(:default, :max, [100.0]),
-        :mean => VariableEstimate(:default, :mean, [50.0]),
-        :modefit => VariableEstimate(:default, :modefit, [75.0]))
-    #update
-    mergeUpdateVariableSolverData!(dfg, newvar)
-    #TODO maybe implement ==; @test newvar==var
-    Base.:(==)(varest1::VariableEstimate, varest2::VariableEstimate) = begin
-        varest1.lastUpdatedTimestamp == varest2.lastUpdatedTimestamp || return false
-        varest1.ppeType == varest2.ppeType || return false
-        varest1.solverKey == varest2.solverKey || return false
-        varest1.estimate == varest2.estimate || return false
-        return true
+    if VARTYPE == DFGVariableSummary
+        global dfg
+        #get the variable
+        var = getVariable(dfg, :a)
+        #make a copy and simulate external changes
+        newvar = deepcopy(var)
+        estimates(newvar)[:default] = Dict{Symbol, VariableEstimate}(
```

Comment:
would probably have to change to `AbstractPointParametricEst` ?

Reply:
Jip, but both in branches. I think one of them needs to be merged first, then it can be updated. To get travis to pass

Reply:
suggestion: maybe merge the other PR to master first, then pull master in here and get tests to pass here and finally merge this PR after?

Reply:
im guessing #183

Reply:
Jip

Reply:
Okay so first 183? We can merge that this weekend :+1:

Reply:
Merged and updated, let me know if I missed anything.

Reply:
LGTM